### ISSUE TITLE
Chrome Client Issue on iOS

### ISSUE THREAD

**Issue by senyah** [suma-project/Suma#61]
I recently set up Suma at my school and ran into a Web SQL issue on Chrome.

After some searching, it looks like Chrome for iOS switched to WKWebView in version 48 and it doesn't work with Web SQL. The bug report is located at https://bugs.webkit.org/show_bug.cgi?id=137760. Here is a screenshot of the error:

<img width="511" alt="chrome error" src="https://cloud.githubusercontent.com/assets/18532387/14605866/9c8462ba-0548-11e6-94fd-68428926f8bd.png">

**Comment by cazzerson** [comment]
Yikes...could you tell me what version of iOS you are using? We'll try it
out here. Also, could you try going into the iOS preferences and resetting
the Safari cache and databases?

Thanks,
Jason

**Comment by senyah** [comment]
I edited my initial post just minutes after submitting it because I also encountered the empty initiative selection on Dolphin. Dolphin was a caching issue. I realized that a previous user of Safari set it to open in private browsing so once I disabled it everything started working. 

The only issue at this point is Chome, but it's not a show stopper since Safari works when not in private browsing.

**Comment by cazzerson** [comment]
OK, great. I'll close this for now and keep an eye out on Chrome. Third party browsers on iOS have always been an oddity, since they haven't been able to use their own engines.

**Comment by AntBand** [comment]
This issue has reared its head again, and not just with Chrome. I think Apple has done something with its recent updates. The Web SQL error suddenly began showing up with Safari and Chrome (77.3865.103) on iOS (13.1.2) and Safari (13.0.2 )on the desktop (OS 10.14.6). I've tried clearing all app data to no avail.

**Comment by cazzerson** [comment]
Unfortunately, we just discovered that the iOS browser engine has disabled Web SQL, which is required for the client to function. We're working on writing this up but I'll keep this issue open until then. The two main options:

1) Use an Android device AND Chrome or Firefox instead.
2) Reenable Web SQL in iOS Safari following the instructions at the bottom of this page: https://blog.appstudio.dev/2019/09/the-future-of-sqlite/

We have confirmed at UNC at Chapel Hill that option #2 will work.
